### DISCUSSION TITLE
Airbyte Connector Builder UI: 3 Success Stories

### DISCUSSION THREAD

**Discussion by masyagin1998** [airbytehq/airbyte#29887] [Show and tell]
Hey everyone!
My name is Mikhail Masyagin an I am a Senior Software Engineer at [Anecdote](https://www.anecdoteai.com/).
Here I would like to share my blog post about Airbyte No-Code & Low-Code Sources development. In this article I provide a complete step-by-step guide on rewriting existing Python CDK sources to Low-Code CDK via Connector Builder UI.

The following topics are considered:

- **IEEE** rules for rewriting connectors to Low-Code CDK.
- Basic source conversion from Python to Low-Code CDK via Connector Builder UI.
- Advanced source conversion with combo of both Low-Code and Python CDKs.
- Advance source conversion with story about tricking Airbyte CDK's paginator.

Feel free to check it out here: https://masyagin1998.hashnode.dev/airbyte-connector-builder-ui-3-success-stories

I am available for messages and will always respond to your comments.

Looking forward to engage with you all. 🚀🚀🚀

Hey! You! Yes, you! You are amazing!❤️

**Comment by dmitrii-mitrofanoff** [comment]
Intresting.
